cypress/e2e/views/migration-wave.view.ts

```diff
@@ -10,6 +10,7 @@ export enum MigrationWaveView {
     stakeHolderGroupsInput = "#stakeholder-groups-toggle-select-multi-typeahead-typeahead",
     actionsButton = "button[aria-label='Actions']",
     yearInput = "input[aria-label='Select year']",
+    fieldHeader = ".pf-c-table__sort",
```

Comment:
Migration wave table has 
"aria-label="Migration waves table"

Instead of fieldHeader Please use this 

export const migrationWavesTable = "table[aria-label='Migration waves table']";